option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message GetCardAttrRequest
{
 repeated U64Type cardId = 1;
}
